src/CADL.Extension/Emitter.Csharp/src/emitter.ts

```diff
@@ -8,6 +8,7 @@ import {
     getServiceTitle,
     getServiceVersion,
     getSummary,
+    ModelType,
```

Comment:
Is this import required? It seems it's not used.

Reply:
Fixed. Cleaned up other unused imports